src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -332,15 +335,31 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
         const isCopyRoutePathSegmentViewVisible =
             copySegmentStore!.startNode && copySegmentStore!.endNode;
         const isSavingPrevented = ENVIRONMENT === 'prod' || ENVIRONMENT === 'stage';
-
         return (
             <div className={s.routePathView}>
-                <RoutePathHeader
-                    routePath={routePathStore!.routePath!}
-                    isNewRoutePath={this.props.isNewRoutePath}
-                    isEditing={!routePathStore!.isEditingDisabled}
-                    onEditButtonClick={routePathStore!.toggleIsEditingDisabled}
-                />
+                <div className={s.sidebarHeaderSection}>
+                    <SidebarHeader
+                        isEditButtonVisible={!this.props.isNewRoutePath}
+                        onEditButtonClick={routePathStore!.toggleIsEditingDisabled}
+                        isEditing={!routePathStore!.isEditingDisabled}
+                    >
+                        {this.props.isNewRoutePath
+                            ? 'Uusi reitinsuunta'
+                            :
+                            <div className={s.topic}>
+                                <div className={s.link} title={`Avaa linkki ${routePath.lineId}`} onClick={() => NavigationUtils.openLineView(routePath.lineId)}>{routePath.lineId}</div>
```

Comment:
Since the title is now interactive thing I think it could be colored with the corresponding color (green for trams, blue for busses etc.) to make it "pop"

Reply:
I have a task about making a generic component of line & route link: https://github.com/HSLdevcom/jore-map-ui/issues/1263

Added your comment to the task